[2025-02-26T14:21:34.496+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-26T14:21:34.529+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [queued]>
[2025-02-26T14:21:34.546+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [queued]>
[2025-02-26T14:21:34.547+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-26T14:21:34.575+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2022-01-23 00:00:00+00:00
[2025-02-26T14:21:34.594+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2022-01-23T00:00:00+00:00', '--job-id', '519', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpcpe0e2g3']
[2025-02-26T14:21:34.595+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6131) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-26T14:21:34.596+0000] {standard_task_runner.py:105} INFO - Job 519: Subtask first_task
[2025-02-26T14:21:34.597+0000] {standard_task_runner.py:72} INFO - Started process 6197 to run task
[2025-02-26T14:21:34.674+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [running]> on host 25887b48f6fb
[2025-02-26T14:21:34.838+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2022-01-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-23T00:00:00+00:00'
[2025-02-26T14:21:34.841+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-26T14:21:34.841+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-26T14:21:34.842+0000] {logging_mixin.py:190} INFO - Current task name:first_task state:running start_date:2025-02-26 14:21:34.530252+00:00
[2025-02-26T14:21:34.843+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag and current dag run status:running
[2025-02-26T14:21:34.843+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-26T14:21:34.845+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-26T14:21:34.846+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2025-02-26T14:21:34.889+0000] {subprocess.py:99} INFO - Output:
[2025-02-26T14:21:34.894+0000] {subprocess.py:106} INFO - hello world, this is the first task!
[2025-02-26T14:21:34.896+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-26T14:21:34.950+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-26T14:21:34.951+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, run_id=scheduled__2022-01-23T00:00:00+00:00, execution_date=20220123T000000, start_date=20250226T142134, end_date=20250226T142134
[2025-02-26T14:21:35.159+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-26T14:21:35.160+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-26T14:21:35.160+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag queued_at:2025-02-26 14:21:32.252956+00:00
[2025-02-26T14:21:35.160+0000] {logging_mixin.py:190} INFO - Task hostname:25887b48f6fb operator:BashOperator
[2025-02-26T14:21:35.180+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-26T14:21:35.240+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-02-26T14:21:35.249+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
